pkgs/applications/science/molecular-dynamics/lammps/default.nix

```diff
@@ -67,8 +61,8 @@ stdenv.mkDerivation rec {
       under the terms of the GNU Public License (GPL).
       '';
     homepage = http://lammps.sandia.gov;
-    license = stdenv.lib.licenses.gpl2;
-    platforms = stdenv.lib.platforms.linux;
-    maintainers = with lib.maintainers; [ costrouc ];
+    license = licenses.gpl2;
+    platforms = [ "x86_64-linux" ];
```

Comment:
The previous version actual did build on `aarch64`. I guess we could leave it as `platforms.linux` and see if it builds on arm.